src/main/java/Duke.java

```diff
@@ -6,5 +8,64 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        String dashes = "____________________________________________________________";
```

Comment:
Perhaps this would be better as a constant (final, with appropriate naming style)?